java/README.md

```diff
@@ -0,0 +1,146 @@
+Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All Rights Reserved. SPDX-License-Identifier: 0BSD
+
+# Sample code
+
+**This repo contains sample code to help you get started with Daml. Please bear
+in mind that it is provided for illustrative purposes only, and as such may not
+be production quality and/or may not fit your use-cases. You may use the
+contents of this repo in parts or in whole according to the BSD0 license:**
+
+> Copyright © 2024 Digital Asset (Switzerland) GmbH and/or its affiliates
+>
+> Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted.
+>
+> THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+> **This repository does not accept Pull Requests at the moment.**
+
+# Example Jvm Bot
+
+This repository has a simple skeleton example of a Jvm Automations that can be run in Daml Hub. This can be copied and then adjusted to your application.
```

Comment:
```suggestion
This repository has a simple skeleton example of a JVM automation that can be run in Daml Hub. This can be copied and then adjusted to your application.
```